cooltools/api/snipping.py

```diff
@@ -198,7 +230,7 @@ def _pileup(data_select, data_snip, arg):
     return np.dstack(stack), feature_group["_rank"].values
 
 
-def pileup_legacy(features, data_select, data_snip, map=map):
+def _pileup(features, data_select, data_snip, map=map):
```

Comment:
it would be good to add to the docstring that this is the internal function, which assumes properly formatted inputs, taht are handled by the user-facing function `pileup`. not sure the best way to phrase

Reply:
as well as a bit more about what it does beyond handling on & off diagonal cases

Reply:
It's private now, so I think these things are all assumed?..